### PR TITLE
ALC-316 transaction status & ALC-190 Style Swap Confirmation Modal

### PR DESCRIPTION
**Pull request by juice0x**
- Added react-toastify for toast messages.
- notify user when status of transaction changes (Pending / failed/ success)
- wrap/unwrap state update also works and uses simliar toast
- update swap confirmation modal style

### CODE REVIEW

File: src/websocket/index.ts [hunk 1]
```diff
@@ -39,14 +45,24 @@ export interface TransactionReq {
 }
 
 export interface TransactionRes {
-  serializedSwap: string
-  serializedApprove: string | undefined
+  transaction: TransactionProcessed
   status: string
```
Comment (Dire-0x) [inline src/websocket/index.ts]: Should this use the `Status` enum instead of `string` ?

File: src/websocket/index.ts [hunk 2]
```diff
@@ -93,36 +155,45 @@ export default function Sockets(): null {
...
+      const hash = keccak256(transaction.transaction.serializedSwap)
+
+      const transactionId = {
+        chainId: transaction.transaction.chainId,
+        hash
+      }
+
+      switch (transaction.status) {
+        case Status.FAILED_TRANSACTION:
+          removeTransaction({
```
Comment (Dire-0x) [inline src/websocket/index.ts]: I don't think we should remove the transactions when they have a `FAILED_TRANSACTION` status. The user should see the failed transactions in the transactions state.

File: src/websocket/index.ts (hunk 1 above)
Comment (juice0x) [reply src/websocket/index.ts]: fixed

File: src/websocket/index.ts (hunk 2 above)
Comment (juice0x) [reply src/websocket/index.ts]: sounds good, the flow has been updated to show the failed
 requests

**Review by Dire-0x — APPROVED** [review]
lgtm, I am unable to see the toast messages but lets address the toast in another issue/PR

**Pull request merged**